inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -137,6 +144,9 @@ public function insert_used_css( array $data ) {
 			return false;
 		}
 
+		// Save used_css into filesystem.
+		$this->save_used_css_in_filesystem( $data['url'], $data['css'], $data['is_mobile'] );
```

Comment:
Why not move the `save_used_css_in_filesystem()` in `save_or_update_used_css()` ?

Reply:
I did that to only save into filesystem in case the field is successfully inserted or successfully updated and to do that at `save_or_update_used_css` I will change it a bit to check the response from insert/update methods and then call it again twice.